### PR TITLE
Home task 01_HTML ... 05_HTML

### PR DESCRIPTION
**Pull request by mgolubenkov**

### CODE REVIEW

**Comment by mgolubenkov** [comment]
16*11*2018

File: mgolubenkov/01_HTML/Python1.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+		<HEAD>
+			<META charset="utf-8">
+			<TITLE>Программа на Python, Часть 1</TITLE>
+		</HEAD>
+		<BODY> 
+			<H1>HOW-TO: Программа на Python, Часть 1</H1>
+			<a herf="#chap_1">Программа на Python — часть 1</a> <br>
+			<a herf="#chap_2">Программа на Python — часть 2</a> <br>
+			<a herf="#chap_3">Программа на Python — часть 3</a> <br>
+UUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUU
```
Comment (pluhin) [inline mgolubenkov/01_HTML/Python1.html]: Need to remove this line

File: mgolubenkov/01_HTML/Python1.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+	<HTML>
+		<HEAD>
+			<META charset="utf-8">
+			<TITLE>Программа на Python, Часть 1</TITLE>
+		</HEAD>
+		<BODY> 
+			<H1>HOW-TO: Программа на Python, Часть 1</H1>
+			<a herf="#chap_1">Программа на Python — часть 1</a> <br>
```
Comment (pluhin) [inline mgolubenkov/01_HTML/Python1.html]: here should be a list

File: mgolubenkov/01_HTML/html0.html [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```
Comment (pluhin) [inline mgolubenkov/01_HTML/html0.html]: This task is not finished

File: mgolubenkov/02_HTML/my_CALENDAR.html [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+			vsides		Рисовать только вертикальные границы (слева и справа от таблицы).
+			rhs			Граница только на правой стороне таблицы.
+			lhs			Граница только на левой стороне таблицы.-->
+  		<table border="1" align="center" width="14%">
+   		<caption><b>My manth MARCH</b></caption>
+   		<tr>
+    		<th colspan="6">Март</th>
+   		</tr>
+   		<tr>
+    		<th><b>ПН</b></th>
```
Comment (pluhin) [inline mgolubenkov/02_HTML/my_CALENDAR.html]: Could you please remove all b tags from all th tags, th tag already provide bold font

File: mgolubenkov/03_HTML/Piton01.html [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+		<HEAD>
+			<META charset="utf-8">
+			<TITLE>main</TITLE>
+		</HEAD>
+		<BODY>
+			<H2>main.mtml</H2>
+			<P>	<DEL>Добрый день, 
+				<EM>Уважаемые слушатели</EM>
+				</DEL>
+
```
Comment (pluhin) [inline mgolubenkov/03_HTML/Piton01.html]: This task is not finished

File: mgolubenkov/04_HTML/forms1.html [hunk 6]
```diff
@@ -0,0 +1,180 @@
+<!DOCTYPE html>
```
Comment (pluhin) [inline mgolubenkov/04_HTML/forms1.html]: This file forms, does not contain from tags at all

File: mgolubenkov/04_HTML/forms1.html [hunk 7]
```diff
@@ -0,0 +1,180 @@
...
+							<input type="radio" name="search_dist" value="google">
+							в Google
+						</label><br>
+						<i>Длина запроса не более 10 символов</i>
+                </fieldset>
+				</td>
+				<!--01-2-->
+				<td>
+				<fieldset>
+					<p><b>Простая форма поиска с вариантами:</b><br></p>
```
Comment (pluhin) [inline mgolubenkov/04_HTML/forms1.html]: This should be header

File: mgolubenkov/04_HTML/forms1.html [hunk 8]
```diff
@@ -0,0 +1,180 @@
...
+					<input type="checkbox" name="drink" value="rad1">Оставаться в системе<Br>
+					<input type="submit" value="Войти"><br>
+					<i>* - обязательное поле</i><br>
+				</fieldset>	
+				</td>
+			<!--02-2-->
+				<td>
+				<fieldset>
+						<legend>Как вам?</legend>
+						<input type="radio" name="drink" value="rad1" checked>ничего<Br>
```
Comment (pluhin) [inline mgolubenkov/04_HTML/forms1.html]: All texts should be in label

File: mgolubenkov/04_HTML/forms1.html [hunk 9]
```diff
@@ -0,0 +1,180 @@
...
+            	<td><b>Отзыв:</b></td>
+				<td><b>Выпадающий и мульти списки:</b></td>
+			</tr>
+			<tr>
+				<td>
+				<fieldset>
+					<legend>Оставьте Ваш отзыв</legend>
+						<input type="text" name="search" maxlength="10">
+						<span>*</span> Имя
+						<input type="text" name="search" maxlength="10">
```
Comment (pluhin) [inline mgolubenkov/04_HTML/forms1.html]: type=email

File: mgolubenkov/04_HTML/forms1.html [hunk 10]
```diff
@@ -0,0 +1,180 @@
...
+			<tr>
+            	<td><b>Отзыв:</b></td>
+				<td><b>Выпадающий и мульти списки:</b></td>
+			</tr>
+			<tr>
+				<td>
+				<fieldset>
+					<legend>Оставьте Ваш отзыв</legend>
+						<input type="text" name="search" maxlength="10">
+						<span>*</span> Имя
```
Comment (pluhin) [inline mgolubenkov/04_HTML/forms1.html]: where is label?

File: mgolubenkov/04_HTML/forms1.html [hunk 11]
```diff
@@ -0,0 +1,180 @@
...
+						<input type="text" name="search" maxlength="10">
+						<span>*</span> e-mail<br>
```
Comment (pluhin) [inline mgolubenkov/04_HTML/forms1.html]: where is label?

File: mgolubenkov/04_HTML/forms1.html [hunk 12]
```diff
@@ -0,0 +1,180 @@
...
+					<td><b>Валидация данных HTML5:</b></td>
+			</tr>
+			<tr>
+					<td>
+					<fieldset>
+						<legend>Upload</legend>
+							Файл:<br>
+							<input type="file" name="file_Fild" size="1" dir="ltr">
+							No file chosen <br>
+							Срок хранения (года):<br>
```
Comment (pluhin) [inline mgolubenkov/04_HTML/forms1.html]: where is label?

File: mgolubenkov/04_HTML/forms1.html [hunk 13]
```diff
@@ -0,0 +1,180 @@
...
+							enctype="multipart/form-data" для отправки
+							файла на сервер</i>
+					</fieldset>	
+					</td>
+		<!--04-2-->
+					<td>
+						<fieldset>
+							<legend>Строка поиска</legend>
+							<input type="text" name="name_03" pattern="[А-Я][а-я]+" placeholder="Только русские буквы"><br>
+							<i>Телефонный номер в формате<br>
```
Comment (pluhin) [inline mgolubenkov/04_HTML/forms1.html]: where is label?

File: mgolubenkov/04_HTML/forms1.html [hunk 14]
```diff
@@ -0,0 +1,180 @@
...
+							+ХХХ-ХХ-ХХХХХХХ, где Х - любое числоы</i><br>
+							<input type="text" name="name_04" pattern="\+375-(29|44|25)-[0-9]{7}" placeholder="+375-29-1234567"><br>
```
Comment (pluhin) [inline mgolubenkov/04_HTML/forms1.html]: wrong pattern

**Comment by pluhin** [comment]
Any updates?

**Pull request closed without merging**